lnbits/static/js/wallet.js

```diff
@@ -225,6 +225,7 @@ new Vue({
         this.parse.data.request = this.parse.data.request.slice(6)
       } else if (req.indexOf('lightning=lnurl1') !== -1) {
```

Comment:
should we also use req.toLowerCase here?

Reply:
the variable was set, probably to avoid writing `this.parse.data.request` a few times... it wasn't used, BTW... 

could this all block code could be optimized? yes...

Reply:
can you check with latest commit, if it looks better? at least i think it reduces a couple lines

Reply:
Yes. Looks much more clean now.